adapters/openx/params_test.go

```diff
@@ -56,7 +59,8 @@ var invalidParams = []string{
 	`{"unit": "123", "delDomain": "foo.b"}`,
 	`{"unit": "123", "delDomain": "foo.barr"}`,
 	`{"unit": "123", "delDomain": ".bar"}`,
-	`{"unit": "123", "delDomain": "foo.bar", "customFloor": "0.1"}`,
+	`{"unit": "123", "delDomain": "foo.bar", "customFloor": ""}`,
+	`{"unit": "123", "delDomain": "foo.bar", "customFloor": "-0.1"}`,
 	`{"unit": "123", "delDomain": "foo.bar", "customFloor": -0.1}`,
 	`{"unit": "123", "delDomain": "foo.bar", "customParams": "foo: bar"}`,
```

Comment:
I recommend adding customFloor tests invalid test cases as "1." and "1.0x" to add more test coverage of the regex.